goreleaser.yml

```diff
@@ -0,0 +1,31 @@
+project_name: trivy_plugin_attest
+builds:
+  -
+    main: main.go
+    binary: attest
+    ldflags:
+      - -s -w
+      - "-extldflags '-static'"
+      - -X main.version={{.Version}}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - darwin
+      - linux
+    goarch:
+      - amd64
+      - arm64
+
```

Comment:
We should have `release` here.
https://goreleaser.com/customization/release/

Reply:
We don't have to add a release section because the default release settings will upload the artifact to the release page.